README.md

```diff
@@ -1 +1,115 @@
-# Spanish_specific_translation_guidelines
\ No newline at end of file
+# Spanish_specific_translation_guidelines
+
+<img src='https://github.com/alan-turing-institute/the-turing-way/blob/main/book/website/figures/logo/logo.png?raw=true' align="right" height="239" /> 
+
+[Alejandro Coca-Castro](https://github.com/acocac), [Camila Rangel Smith](https://github.com/crangelsmith), Pamela Villar González, Reina Camacho Toro
+, [María Nanton](https://github.com/mcnanton)
+
+
+Some inspiration for these guidelines comes from the collaborative translations of [R4DS](https://github.com/cienciadedatos/documentacion-traduccion-r4ds) by [@cienciadedatos](https://github.com/cienciadedatos), [Teaching Tech Together](https://github.com/gvwilson/teachtogether.tech/blob/master/es/README.md) to Spanish, the [R Contributors translations working group](https://contributor.r-project.org/translations/) and the [R OpenSci translation project](https://translationguide.ropensci.org/).
+
+*switch to spanish but feel free to enter in contact in English :)* 
+
+## Código de conducta
+
+Este espacio está ligado a [*The Turing Way*](https://the-turing-way.netlify.app/welcome.html) y su [código de conducta](https://the-turing-way.netlify.app/community-handbook/coc.html) 
+
+## Orientaciones para la traducción
+
+### Contexto
+
+A la hora de realizar traducciones se suelen producir algunos errores recurrentes habitualmente por parte de aquellas personas que no están acostumbradas a realizar esa tarea. 
+Esta sección incluye una serie de guías de recomendaciones con los errores más frecuentes en la traducción de textos técnicos así como un glosario para lograr mayor uniformidad en las traducciones. 
+Se recomienda su uso por todos los traductores que forman parte del esfuerzo de internacionalización al español.
+En general, se debe evitar la traducción directa del inglés. 
+El traductor debe primero leer y entender lo que va a traducir antes de hacerlo y, en muchos casos, es mejor traducir con las palabras que uno utilizaría, y no necesariamente con las mismas que utilizó el que lo escribió en inglés. 
+En programas y aplicaciones se recomienda utilizarlos antes para determinar el uso de los textos en el contexto del programa. 
+Por ejemplo, la cadena Save en un programa podría traducirse por Salvar si se trata de un juego y es una operación para salvar a un compañero de un peligro o Guardar (más común) si se trata de guardar los resultados del programa a un archivo. 
+Una traducción de calidad es aquella que ofrece al lector el mismo significado que el término original aunque no necesariamente con las mismas palabras o frases.
+
+### Normas
+
+A continuación se listan una serie de normas a tener en cuenta a la hora de realizar las traducciones:
+
+#### Tratamiento al lector
+
+Queremos que The Turing Way se sienta como un documento informal y amigable, por lo que excepto casos contados, ha de tratarse al lector siempre de tú. 
```

Comment:
Solemos usar itálicas al referirnos a _The Turing Way_ en inglés, ¿lo mantenemos en español? Además de–obvio–no traducir la expresión aunque crowdin insista con "la vía de Turing"

Reply:
Jajaja. Creo que el itálico ayuda a destacar que se está hablando del libro